src/me/fru1t/subedit/utils/Utils.kt

```diff
@@ -45,13 +45,14 @@ object Utils {
   }
 
   fun askYes(): Boolean = readLine()?.toLowerCase() == "y"
-  fun askForFontSize(): Int {
-    println("Font size?")
+
+  fun askForInt(message: String): Int {
+    println(message)
     val size = readLine()?.toIntOrNull()
     if (size == null) {
       println("Invalid input...")
-      return askForFontSize()
+      return askForInt(message)
     }
     return size
   }
```

Comment:
Changed this to be a more generic function that takes a prompt message as a parameter.